TimerCalculatorUtility/TimerCalculatorUtility/Views/TimerView.xaml

```diff
@@ -0,0 +1,105 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+              xmlns:viewmodels="clr-namespace:TimerCalculatorUtility.ViewModels"
+             xmlns:converters ="clr-namespace:TimerCalculatorUtility.Converters"
+             x:Class="TimerCalculatorUtility.Views.TimerView"
+             IconImageSource="humburger.png"
+             Title="Stopwatch">
+    
+    <ContentPage.Resources>
+        <ResourceDictionary>
+            <converters:ListViewVisibilityConverter x:Key="ListViewVisibilityConverter"/>
+            <converters:StartButtonVisibilityConverter x:Key="StartButtonVisibilityConverter"/>
+            <converters:LapButtonVisibilityConverter x:Key="LapButtonVisibilityConverter"/>
+            <converters:ResumetButtonVisibiliryConverter x:Key="ResumetButtonVisibiliryConverter"/>
+            <converters:ResetButtonVisibilityConverter x:Key="ResetButtonVisibilityConverter"/>
+            <converters:StopButtonVisibilityConverter x:Key="StopButtonVisibilityConverter"/>
+            <Color x:Key="PinkButton">#d4526a</Color>
+            <Color x:Key="BlueButton">#6063f0</Color>
+        </ResourceDictionary>
+    </ContentPage.Resources>
+    
+    <ContentPage.BindingContext>
+        <viewmodels:TimerViewModel x:Name="ViewModel"/>
+    </ContentPage.BindingContext>
+    
+    <ContentPage.Content>
+        <AbsoluteLayout>
+            <Label Text="{Binding TimeString}"
+                   FontSize="65"
+                   TextColor="{StaticResource Key=Black}"
+                   AbsoluteLayout.LayoutFlags="All"
+                   AbsoluteLayout.LayoutBounds="0.53,0.17,0.7,0.13"
+                   x:Name="CurrentTimeLabel"/>
+            <Label Text="{Binding LapTimeString}"
+                   FontSize="35"
+                   AbsoluteLayout.LayoutFlags="All"
+                   AbsoluteLayout.LayoutBounds="0.5, 0.3, 0.4, 0.08"/>
+         
+            <ListView ItemsSource="{Binding Times}"
+                      IsVisible="{Binding Path=Status, 
+                                                 Converter={StaticResource ListViewVisibilityConverter}}"
```

Comment:
Line formatting, format properties in one vertical line, so they can be read more easily.